source/Symbiont.test.cc

```diff
@@ -314,18 +314,118 @@ TEST_CASE("Process") {
 
 }
 
+TEST_CASE("WantsToInfect"){
```

Comment:
This test should be in HostSymbiontUnitTest instead since it involves hosts and symbionts, not just a symbiont.